pkgs/by-name/em/emacsclient-commands/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, gnumake
+}:
+
+buildGoModule {
+  pname = "emacsclient-commands";
+  version = "unstable-2023-09-22";
+
+  src = fetchFromGitHub {
+    owner = "szermatt";
+    repo = "emacsclient-commands";
+    rev = "8f5c8a877794ed51f8225036e36fd5ce272b17f3";
+    hash = "sha256-OlcB5VqWYdl0wz1y8nmG6Xgdf5IPOUQ31UG1TDxQAis=";
+  };
+
+  vendorHash = "sha256-8oREed2Igz5UvUTDdOFwW5wQQy3H8Xj8epxo6gqnZFA=";
+
+  buildInputs = [ gnumake ];
+
+  buildPhase = ''
+    runHook preBuild
+    DESTDIR=$out/ make install
```

Comment:
Is make install supposed to be used in build phase?

Reply:
make install also does the go build as far as I can tell so I think it's fine, but I might be wrong

Reply:
@drupol yes, make install also performs the build operations.  using the default build phase also breaks everything since the project uses an unusual directory layout (multiple go packages in one directory).

as far as i can tell `make install` works by building the programs and outputting them directly to DESTDIR (this works because go doesn't produce intermediate build artifacts the same way C does)